okta/requestExecutor.go

```diff
@@ -74,7 +95,83 @@ func (re *RequestExecutor) NewRequest(method string, url string, body interface{
 	if err != nil {
 		return nil, err
 	}
-	req.Header.Add("Authorization", "SSWS "+re.config.Okta.Client.Token)
+
+	if re.config.Okta.Client.AuthorizationMode == "SSWS" {
+		req.Header.Add("Authorization", "SSWS "+re.config.Okta.Client.Token)
+	}
+
+	if re.config.Okta.Client.AuthorizationMode == "PrivateKey" {
+		if re.cache.Has("OKTA_ACCESS_TOKEN") {
+			token := re.cache.GetString("OKTA_ACCESS_TOKEN")
+			req.Header.Add("Authorization", "Bearer "+token)
+		} else {
+			priv := []byte(re.config.Okta.Client.PrivateKey)
```

Comment:
In the config, `PrivateKey` is defined as a string: https://github.com/okta/okta-sdk-golang/pull/112/files#diff-ad4db84096a7b93a350a17acf8082116R49

I'm guessing this works, but is there any need to specify encoding when converting string, to byte[]? Is it assumed to be UTF-8?

Reply:
Nope.   all good here